configure.ac

```diff
@@ -234,34 +234,12 @@ fi
 
 AM_CONDITIONAL(ENABLE_DEBUGGING, [test "x$enable_debugging" = "xyes"])
 
-
 # Checks for programs
 # -------------------
-
 AC_PROG_CC
-AC_PROG_CC_C99
 AC_PROG_RANLIB
 AC_C_BIGENDIAN
 
-# https://www.gnu.org/software/autoconf-archive/ax_prog_cc_for_build.html
-AX_PROG_CC_FOR_BUILD
```

Comment:
This was introduced in #2747 to fix the cross-compilation failure.
Removing this will break it again, I think.